style_guides/react_style_guide.md

```diff
@@ -90,23 +31,6 @@ Bad:
 <react-component name="HelloComponent" props="person" watch-depth="reference" />
 ```
 
-### Prefix ui_framework elements with kui, but not their file names.
```

Comment:
:information_source: We don't have (or rather are about to remove) KUI completely, so I think we don't need style guide rules specific for KUI anymore.